.gitlab/test.yml

```diff
@@ -0,0 +1,33 @@
+test:
+  extends: .check_skip_rust
+  timeout: 2 hours
+  image: paritytech/ci-unified:bullseye-1.77.0
+  stage: test
+  cache:
+    key: cache-test
+    paths:
+      - target/
+      - $HOME/.cargo/bin/
+      - $HOME/.cargo/registry/index/
+      - $HOME/.cargo/registry/cache/
+      - $HOME/.cargo/git/db/
+  script:
+    - time cargo test --all --all-targets --locked
+
+test-features:
+  extends: .check_skip_rust
+  timeout: 2 hours
+  image: paritytech/ci-unified:bullseye-1.77.0
+  stage: test
+  needs:
+    - test
+  cache:
+    key: cache-test
+    paths:
```

Comment:
Is it possible to hide the paths to cache behind a variable or something, so that we avoid repeating them everywhere, with the risk of making them inconsistent over time?

Reply:
Added as env variable, yes it looks cleaner